src/main.c

```diff
@@ -12,15 +12,15 @@
 #include "lvgl.h"
 #include "app_hal.h"
 
-#include "demo.h"
+#include "demos/lv_demos.h"
 
 int main(void)
 {
 	lv_init();
 
 	hal_setup();
 
-	demo_create();
+  lv_demo_widgets();
```

Comment:
```suggestion
	lv_demo_widgets();
```

Reply:
This breaks tests.

Reply:
2 problem in using the demo: 1. Need to add LVGL memory config, otherwise the sim may crash. 2. Need to force include demos folder for compilation using src_filter.